tools/vrt-selector/src/story-candidates.js

```
@@ -0,0 +1,228 @@
+      file === "pnpm-lock.yaml" ||
+      file.startsWith("libs/") ||
+      file.startsWith("tools/vrt-selector/") ||
+      file.match(/^packages\/(ui|web)\/regconfig\.json$/)
+    ) {
+      VRT_PACKAGES.forEach((pkg) => affectedPackages.add(pkg));
+      continue;
+    }
+
+    for (const [prefix, targets] of Object.entries(PACKAGE_DEPS)) {
+      if (file.startsWith(`${prefix}/`)) {
+        targets.forEach((target) => affectedPackages.add(target));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)